package.json

```diff
@@ -16,17 +16,29 @@
       "transform-flow-strip-types"
     ]
   },
+  "eslintConfig": {
+    "extends": "ello",
+    "globals": {
+      "AndroidInterface": false,
+      "ENV": false
```

Comment:
Yeah, this was added in an effort to get `eslint` to run by itself this morning when Matt and I were troubleshooting it. It didn't help one way or another so yeah, lets kill it.